inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -913,7 +940,8 @@ void GNAPlugin::LoadNetwork(CNNNetwork & _network) {
         inputsDesc->getPtrInputsGlobal(input.first).resize(gnaFlags->gna_lib_async_threads_num);
     }
 
-    // CreatingLayer primitives
+    // CreatingLayer primitivestop
```

Comment:
what does primitivestop mean?

Reply:
Oh, this is typo mistake, fixed, thanks!